src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -63,6 +61,8 @@ protected function setUp(): void
         $config = DoctrineTestHelper::createTestConfiguration();
         $config->setRepositoryFactory($this->repositoryFactory);
 
+        SingeIntIdEntityRepository::$result = null;
```

Comment:
I'm always afraid to use static properties in tests, it can lead to complex debugging especially when running tests in random order. Should we reset the value of the property in the `tearDown()` method for example? Just to ensure tests don't leak